tests/test_tools/test_w_trace.py

```diff
@@ -59,6 +56,3 @@ def check_output(self, arg_idx):
             assert list(traj_group.keys()) == expected_groups, "H5 groups ({}) are not as expected ({})".format(
                 list(traj_group.keys()), expected_groups
             )
-
-        diff = H5Diff(self.ref_files[arg_idx], self.outfile)
-        diff.check()
```

Comment:
Do we not need to check against the index anymore? Just double checking the logical change was warranted.

Reply:
Seems like the test was never updated to the newer version with H5Diff. Thanks for noticing this! Now fixed.